examples/common/contour-config.yaml

```diff
@@ -16,4 +16,12 @@ data:
     tls:
       # minimum TLS version that Contour will negotiate
       # minimumProtocolVersion: "1.1"
+    # The following config shows the defaults for the leader election.
+    # leaderelection:
+      # configmap-name: contour
+      # configmap-namespace: heptio-contour
+      # TODO(youngnick) not sure whether to document these tbh.
```

Comment:
Please open an issue for this TODO so we don't forget to revisit this before beta.1 ships.

Reply:
still need that todo

Reply:
As agreed, I've removed these params.